site/docs/webrtc/voice-iw.md

```diff
@@ -0,0 +1,191 @@
+
+Once an answer is received on this transaction, an interconnection is established between WebRTC and Voice.
+
+You are able to decide how the flow of the call will proceed based on Bxml verbs that are chosen on setting up the interconnection. Three call control verbs that can be used are: Transfer, Bridge, and Conference, each of which creates a different application experience
+
+| Programmable Voice Verb | Solution Characteristics summary |
+| --- | --- |
+| Transfer | The simplest application behavior, with limited opportunity to invoke services on the PV call leg. |
+| Bridge | Suitable for single-leg interconnection where features might be required for the PV call leg, with slightly higher complexity required in the application |
+| Conference | Enables multiple participants on the PV side of the interconnection, and a full range of PV features and capabilities to be applied on the OV call leg. Additional costs are incurred for the use of the voice conference bridge. |
+
+These three interaction models are described in greater detail below.
+
+## Transfer
+
+The transfer verb creates a single PV call with two endpoints. You can set one end to WebRTC, and the other for a phone number
+
+![Transfer Verb Model](../../static/img/webrtc-pv-transfer.png "Transfer")
+
+An everyday use case for this method is when people use Bandwidth phone numbers as a proxy. If you run a business and wish to protect your personal number, then transfer would be a good option.
+
+Use of the Transfer Bxml Verb creates a single call that will typically be taken down from end to end when it is complete (or times out waiting for an answer). The call ends if the person you are transferring to does not respond. Also, any modifications to the call will take down the connection between WebRTC and the Voice Network.
+
+In general, the Transfer verb is used in inbound or outbound WebRTC/Voice calls to create the call from scratch when the call happens. Here’s a [general example](https://github.com/Bandwidth-Samples/webrtc-hello-world-ts) of how you can use the Transfer verb:
+
+Bandwidth's Voice API will hit a provisioned callback endpoint when you receive an incoming call
+
+```js
+    app.post("/incomingCall", async (req, res) => { const callId = req.body.callId;
+        console.log(`received incoming call ${callId} from ${req.body.from}`);
+        const participant = await createParticipant("hello-world-ts-phone");
+        calls.set(callId, participant);
+```
+
+This generates the response payload containing the Transfer verb that is sent back to the PV API to transfer the call into the WebRTC session
+
+```js
+const bxml = WebRtcController.generateTransferBxml(participant.token);
```

Comment:
This needs to include the call ID from Voice as the second argument

Reply:
thanks - done.